doc/2/guides/develop-on-kuzzle/api-controllers/index.md

```diff
@@ -233,6 +233,16 @@ The main available properties are the following:
  - `args`: Action arguments
  - `body`: Body content
 
+### Extract parameters from request
+
+<SinceBadge version="auto-version" />
+
+The request object expose methods to extract parameters from the request in a standardized way.
+
+Each of those method will check for the parameter presence and type. In case of a validation failure, corresponding errors will be returned.
```

Comment:
```suggestion
Each of those methods will check for the parameter presence and type. In case of a validation failure, the corresponding API error will be thrown.
```